libid/include/fractalb.h

```diff
@@ -40,10 +40,12 @@ int mandel_bf_per_pixel();
 int julia_bf_per_pixel();
 int julia_bf_fractal();
 int julia_z_power_bf_fractal();
+int burning_ship_bf_fractal();
 BFComplex *cmplx_log_bf(BFComplex *t, BFComplex *s);
 BFComplex *cmplx_mul_bf(BFComplex *t, BFComplex *x, BFComplex *y);
 BFComplex *cmplx_div_bf(BFComplex *t, BFComplex *x, BFComplex *y);
 BFComplex *cmplx_pow_bf(BFComplex *t, BFComplex *xx, BFComplex *yy);
+BFComplex *cmplx_poly_bf(BFComplex *out, BFComplex xx, int degree);
```

Comment:
delete this

Reply:
I couldn't as it is needed for burning_ship.cpp ->  burning_ship_bf_fractal(). This one is not done.